STYLE_GUIDE.md

```diff
@@ -0,0 +1,185 @@
+
+Other than that, we mostly recommend following the [AirBnB JavaScript Style Guide](https://github.com/airbnb/javascript).
+
+##  Browser and device support
+
+We aim to support recent versions of major modern browsers - i.e. Chome, Firefox, Edge, Safari, and Opera, on both MacOS and Windows. Keeping progressive enhancement in mind, we aim for decent mobile support where possible, i.e. nothing should be broken on mobile, but it's okay for mobile/older browsers to have a slightly degraded experience.
+
+## Accessibility
+
+We aim to meet [WCAG 2.1 Level AA](https://www.w3.org/WAI/standards-guidelines/wcag/) standards where possible, while acknowledging that this is a data visualization, so some criteria will be difficult to pass without extraordinary effort. However we will do our best within reason.
+
+Kedro-Viz should be navigable with different input devices (e.g. mouse, keyboard, and touchscreen), with obvious and visible focus states on all interactive elements. Text elements should have sufficient colour contrast and font-size to ensure readability to a minimum Level AA standard. Use [WAI-ARIA](https://www.w3.org/WAI/standards-guidelines/aria/) features where applicable to improve screenreader support.
+
+We recommend checking [axe](https://www.deque.com/axe/) and [Lighthouse](https://developers.google.com/web/tools/lighthouse) to review for potential accessibility errors if unsure.
+
+## Semantic HTML
+
+Ensure that page content is marked up with [semantic HTML elements](https://html.com/semantic-markup/) in order to ensure that it is as usable and accessible as possible.
+
+Avoid:
+```jsx
+<div className="form">
+  <div>Contact us</div>
+  <div>Use this form to send us a message.</div>
+  <div>Email address:</div>
+  <input type="text" />
+  <div onClick={handleSubmit}>Submit</div>
+</div>
+```
+Prefer:
+```jsx
+<form onSubmit={handleSubmit}>
+  <h1>Contact us</h1>
+  <p>Use this form to send us a message.</p>
+  <label htmlFor="email">Email address:</label>
+  <input id="email" type="email" />
+  <button>Submit</button>
+</form>
+```
+
+## CSS
+
+### Class names
+
+This project uses the [BEM naming convention](http://getbem.com/), to avoid conflicts, reduce rule specificity, and make classes easier to read and understand. [Reducing rule specificity](https://css-tricks.com/strategies-keeping-css-specificity-low/) is particularly important for improving CSS maintainability, so we try to avoid nesting CSS selectors as much as possible.
+
+We use the `pipeline-` prefix for most HTML/CSS classes, to reduce the risk of clashes in the global namespace when this project is imported into other applications. It's important never 
```

Comment:
"It's important never" is missing its ending

Reply:
Thanks! Yeah this part is very much a draft - it's practically scribbled notes. I didn't expect you to get this much into the weeds of reviewing it! 🙈